src/models/tech-record-document.ts

```diff
@@ -0,0 +1,28 @@
+import {DynamoDbImage, parseStringArray} from "../services/dynamodb-images";
+import {parseTechRecords, TechRecords} from "./tech-record";
+
+// https://wiki.dvsacloud.uk/pages/viewpage.action?spaceKey=HVT&title=Technical+API+Changelog
```

Comment:
We could be looking at automatically generating the interfaces from the specs in this shared module

Reply:
As agreed on chat with Jon, leaving this for a future PR and resolving for now - @SanjeetBJSS accepts the tech debt